pages/index.js

```diff
@@ -1,29 +1,54 @@
 import PlantCard from "@/components/PlantCard";
+import PlantFilter from "@/components/PlantFilter";
 import styled from "styled-components";
 
 export default function HomePage({
   plants,
   onToggleOwned,
   ownedPlantsIds,
   onAddPlants,
+  onFilterPlants,
+  activeFilter,
+  onClearFilter,
 }) {
+  if (plants.length === 0) {
+    return (
+      <>
+        <p>No plants match the filter criteria!</p>
+        <button
+          onClick={() => {
+            onClearFilter()
+          }}
+        >
```

Comment:
```suggestion
        <button onClick={onClearFilter}>
```

Use arrow functions only if:
You need to pass arguments.
You need extra logic inside the handler.

Otherwise, always prefer the direct reference. 
It’s faster & more readable